tests/e2e/basic/_basic_test_setup/setup_e2e_vlan.py

```diff
@@ -0,0 +1,181 @@
+import pytest
```

Comment:
Documentation

Reply:
I have started to write readmes, for each testbed environment, that will explain the setup needed for test cases. i will update here once i complete